go/teams/member_set.go

```diff
@@ -134,6 +134,17 @@ func loadUPAK2(ctx context.Context, g *libkb.GlobalContext, uid keybase1.UID, fo
 	return upak, err
 }
 
+func loadUPAKLite(ctx context.Context, g *libkb.GlobalContext, uid keybase1.UID, forcePoll bool) (ret *keybase1.UpkLiteV1AllIncarnations, err error) {
+	defer g.CTrace(ctx, fmt.Sprintf("loadUPAKLite(%s)", uid), func() error { return err })()
+
+	arg := libkb.NewLoadUserArg(g).WithNetContext(ctx).WithUID(uid).WithPublicKeyOptional()
+	if forcePoll {
+		arg = arg.WithForcePoll(true)
+	}
+	upak, err := g.GetUPAKLoader().LoadLite(arg)
```

Comment:
i can't remember exactly, but i dont think the lite loading really respects the forcePoll request. or maybe just not as much.

Reply:
I don't think it does at all until we do caching